.env

```diff
@@ -0,0 +1,6 @@
+PORT = 5000
```

Comment:
You shouldn't commit .env, only .env.sample. Let's do it like in real project.

Reply:
yeah, TU :+1: